lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1171,8 +1171,7 @@ class ModuleEmitter : public EmitterBase {
   void emitBind(BindOp op);
   void emitBindInterface(BindInterfaceOp op);
 
-  StringRef getNameRemotely(Value value, const ModulePortInfo &modulePorts,
-                            HWModuleOp remoteModule);
+  StringRef getNameRemotely(Value value);
```

Comment:
I think this is unused now (AFAICT its definition has been removed)?

`ExprEmitter::visitSV(XMRRefOp)` says it shares a lot of code with this method-- I think it maybe can also just grab names off the values, WDYT?

Reply:
Ah, I forgot to remove the declaration:) Thanks!